### PR TITLE
Blog/home page update

### PR DESCRIPTION
**Pull request by mtnbarreto**
clean up blog UI
=====================
• added category to each blogpost, main list and in a particular post.
• added writter to main list
• fixed featured post layout issue, text no longer overlaps images. 
• spacing and font updated in titles and content so it's more pleasant to read.

### CODE REVIEW

**Comment by bryant1410** [comment]
title and first post seem to be too next to each other IMHO. The rest looks good

image

**Comment by bryant1410** [comment]
some code still looks big IMO:

image

**Comment by bryant1410** [comment]
can you explain the main changes?

**Comment by bryant1410** [comment]
and doesn't it look too serious in serif font titles?

File: _sass/_fonts.scss [hunk 1]
```diff
@@ -1,2 +1,22 @@
-@import url(//fonts.googleapis.com/css?family=Lato:300,400,700,900,300italic,400italic);
-@import url(//fonts.googleapis.com/css?family=Helvetica%20Neue:300,400,700,900,300italic,400italic);
+// @import url(//fonts.googleapis.com/css?family=Lato:300,400,700,900,300italic,400italic);
+@import url(//fonts.googleapis.com/css2?family=Libre+Baskerville:ital,wght@0,400;0,700;1,400);
+// font-family: 'Libre Baskerville', serif;
+
+// @font-face {
```
Comment (matir91) [inline _sass/_fonts.scss:5]: take a look at it

File: _config.yml [hunk 2]
```diff
@@ -28,6 +29,11 @@ permalink: /:year/:month/:day/:title/
 # sass:
 #   style: :compressed
 
+# Pagination
+paginate: 6
+paginate_path: "/page/page:num/"
+
```
Comment (matir91) [inline _config.yml:35]: rm

**Pull request merged**